ksqldb-test-util/src/main/java/io/confluent/ksql/test/util/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -246,6 +246,7 @@ public String bootstrapServers(final SecurityProtocol securityProtocol) {
    *
    * @return base set of producer properties.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
Why are you reducing `retries` from `MAX_VALUE` to `10`? Are you trying to bound the test time in case of failure? Instead of reducing `retries` you should set `delivery.timeout.ms` for this case.